webui_runner/plugs/lucet_app/shootout/build.sh

```diff
@@ -23,37 +24,20 @@ cp -r ${SIGHTGLASS_ROOT}/benchmarks/shootout/* .
 
 
 #Build shootout
-for cfile in ./*.c; do 
+for cfile in ./*.c; do
     echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -c $cfile -o $(basename -s .c "$cfile").wasm.o
     ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} -c $cfile -o $(basename -s .c "$cfile").wasm.o
 
-    echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm  -nostartfiles -Wl,--no-entry -Wl,--export-all -Wl,--gc-sections 
-          ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm -nostartfiles -Wl,--no-entry -Wl,--export-all -Wl,--gc-sections
+    echo ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm  -nostartfiles
+    ${WASM_CC} ${WASM_ENTRY} ${WASI_CFLAGS} ${WASI_NO_SUPPORT} ${COMMON_CFLAGS} $(basename -s .c "$cfile").wasm.o -o $(basename -s .c "$cfile").wasm -nostartfiles
 
-    echo ${LUCETC} ${LUCETC_FLAGS} --emit=obj  \
-		--bindings=${LUCET_BINDINGS} \
-		--builtins=${LUCET_LIBBUILTINS} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm
-    ${LUCETC} ${LUCETC_FLAGS} --emit=obj  \
-		--bindings=${LUCET_BINDINGS} \
-		--builtins=${LUCET_LIBBUILTINS} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm
-
-
-    echo ${CC} -rdynamic -shared \
-		-L $(dirname ${LUCET_LIBBUILTINS}) \
-		-Wl,-rpath $(dirname ${LUCET_LIBBUILTINS}) \
-		$(basename -s .c "$cfile").wasm -lbuiltins -o $(basename -s .c "$cfile").wasm.so
-
-        ${CC} -rdynamic -shared \
-		-L $(dirname ${LUCET_LIBBUILTINS}) \
-		-Wl,-rpath $(dirname ${LUCET_LIBBUILTINS}) \
-		$(basename -s .c "$cfile").wasm -lbuiltins -o $(basename -s .c "$cfile").wasm.so
+	echo ${LUCETC} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm.so
+	${LUCETC} $(basename -s .c "$cfile").wasm -o $(basename -s .c "$cfile").wasm.so
 done
 
-
-
 #Build implementation.so
 mkdir -p ${SCRIPT_LOC}/bin; cd ${SCRIPT_LOC}/bin
-echo ${CC} ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=${LUCET_APP_ROOT}/build/lucet/target/release/lucet-wasi -shared -o implementation.so ../wrapper.c 
-${CC} -fPIC ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=${LUCET_APP_ROOT}/build/lucet/target/release/lucet-wasi -shared -o implementation.so ../wrapper.c 
+echo ${CC} ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=/opt/lucet/bin/lucet-wasi -shared -o implementation.so ../wrapper.c
+${CC} -fPIC ${COMMON_CFLAGS} -DWORKLOAD_LOCATION=${SCRIPT_LOC}/benchmark -DVM_LOCATION=/opt/lucet/bin/lucet-wasi -shared -o implementation.so ../wrapper.c
```

Comment:
Why not use the variable? Or is this script only intended to be used from within a Docker container?

Reply:
Yes, all of these webui_runner files are written for the container .. but this still should run outside of the container. That said, LUCETC was already defined so no reason can't have a LUCET variable as well so I've added it.